<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q7ToolsWindow</class>
 <widget class="QWidget" name="Q7ToolsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>364</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>364</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>364</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../R/Res.qrc">
    <normaloff>:/images/icons/cgSpy.gif</normaloff>:/images/icons/cgSpy.gif</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="diff">
        <attribute name="title">
         <string>Diff</string>
        </attribute>
        <widget class="QGroupBox" name="gdiff">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>5</y>
           <width>541</width>
           <height>266</height>
          </rect>
         </property>
         <property name="title">
          <string>All tree identifications use view number</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>25</x>
            <y>25</y>
            <width>58</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Version A</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cAncestor">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>45</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="cVersionA">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>45</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>25</y>
            <width>58</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Version B</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="cShowA">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>235</x>
            <y>45</y>
            <width>176</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Show result on B view</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="bDiff">
          <property name="geometry">
           <rect>
            <x>395</x>
            <y>45</y>
            <width>25</width>
            <height>25</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../R/Res.qrc">
            <normaloff>:/images/icons/operate-execute.gif</normaloff>:/images/icons/operate-execute.gif</iconset>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="split">
        <attribute name="title">
         <string>Split</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="merge">
        <attribute name="title">
         <string>Merge</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="process">
        <attribute name="title">
         <string>Process</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bInfo">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../R/Res.qrc">
         <normaloff>:/images/icons/help-view.gif</normaloff>:/images/icons/help-view.gif</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../R/Res.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
